lib/openshift-rest-client.js

```diff
@@ -92,7 +92,7 @@ async function openshiftClient (settings = {}) {
     }
   }
 
-  const client = new Client({ config: kubeconfig || config.fromKubeconfig(), spec, getNames: getNames });
+  const client = new Client({ config: kubeconfig, spec, getNames: getNames });
```

Comment:
This seems to only to take into account if you are not using a custom configuration.  https://github.com/godaddy/kubernetes-client/blob/master/merging-with-kubernetes.md#client-config-